cumulus/parachains/runtimes/bridge-hubs/common/Cargo.toml

```diff
@@ -30,6 +30,7 @@ std = [
 	"sp-runtime/std",
 	"sp-std/std",
 	"xcm/std",
+	"sp-core/std"
```

Comment:
I believe these features should be ordered alphabetically.

Reply:
you just need to use this: https://github.com/paritytech/polkadot-sdk/blob/master/substrate/docs/STYLE_GUIDE.md#manifest-formatting

```
zepter format features
```